content/1.7/back-office/quick-access/quick-access-listing.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Quick access listing
+weight: 1
+---
+
+# Quick access listing
+
+The quick access drop-down, displayed at the top left of every page of the PrestaShop back office lists all the pages added as quick access. 
+It's possible to add the current back office page to the quick access by clicking on "Add current page to QuickAccess". It's also possible to see the list of all the quick access by clicking on "Manage quick accesses".
+
```

Comment:
It's possible to remove the current back office page from the quick access list by clicking on "Remove from Quick Access"
![image](https://user-images.githubusercontent.com/89604350/149144942-287e9ae6-5102-4ca4-98aa-047341bccbf1.png)